src/test/java/automationpractice/com/pages/AddressPage.java

```diff
@@ -33,6 +31,14 @@ public void clickDeleteButton(final String addressName) {
 
     public void acceptAlert() {
         getDriver().switchTo().alert().accept();
+    }
 
+    public void removeAddressFromAddressPage(String addressName){
+        List<String> addressNameList = getAddressNameList();
+        for (String name : addressNameList){
+            if (name.toUpperCase().equals(addressName.toUpperCase())){
+                clickDeleteButton(addressName);
+                acceptAlert();            }
+        }
     }
 }
```

Comment:
Tutaj nie powinieneś sprawdzac czy istnieje taki adres (to możesz zrobić w warstwie wyzej) tylko po prostu usunac adres o nazwie